.eslintrc.js

```diff
@@ -17,21 +16,20 @@ module.exports = {
       "flowtype"
     ],
     "rules": {
-        "indent": [
-            "error",
-            2
-        ],
-        "linebreak-style": [
-            "error",
-            "unix"
-        ],
-        "quotes": [
-            "error",
-            "single"
-        ],
-        "semi": [
-            "error",
-            "always"
-        ]
+      "no-magic-numbers": [
+        "off"
```

Comment:
Yupp, disallowing "magic" numbers doesn't make a lot of sense in the context of this project, since 99% of the time the number is just describing an animation. It wouldn't make sense to store them in variables.